lib/matplotlib/widgets.py

```diff
@@ -3134,6 +3275,20 @@ def extents(self, extents):
         self.set_visible(self.visible)
         self.update()
 
+    @property
+    def rotation(self):
+        """Rotation in degree in interval [0, 45]."""
+        return np.rad2deg(self._rotation)
+
+    @rotation.setter
+    def rotation(self, value):
+        # Restrict to a limited range of rotation [0, 45] to avoid changing
+        # order of handles
+        if 0 <= value and value <= 45:
```

Comment:
I assume that this results in the rotation not following the mouse beyond 45°.

This restriction should be mentioned in the what's new entry. Ideally also somewhere in the regular API docs (maybe at state_modifier_keys?) Otherwise, we'll get bug reports on this.